renovate.json

```diff
@@ -7,5 +7,9 @@
       "packageNames": ["node"],
       "enabled": false
     }
-  ]
+  ],
+  "prConcurrentLimit": 5,
+  "schedule": ["before 9am on monday"],
+  "stabilityDays": 3,
+  "dependencyDashboard": true
```

Comment:
I enabled this setting so that we have visibility into which dependency updates are pending and for which reason. I think it will allow us to say "create this PR even outside the scheduled time" or that kind of thing.

I didn't enable the `dependencyDashboardApproval` setting, so that means that it will still create PR's even before checking them off in the dependency issue.